<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome to your admin account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Search profiles:</string>
   </property>
   <widget class="QRadioButton" name="radioButtonDoc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Doctor Account</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNurse">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nurse Account</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPatient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Patient Account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
